src/backend/common/helpers/tests/data/2023mijac_district_points.json

```diff
@@ -66,9 +66,9 @@
     "frc4381": {
       "alliance_points": 16,
       "award_points": 8,
-      "elim_points": 26,
+      "elim_points": 25,
       "qual_points": 20,
-      "total": 70
+      "total": 69
```

Comment:
I am so confused based on https://frc-events.firstinspires.org/2023/MIJAC/rankings#district and the https://frc-events.firstinspires.org/2023/district/FIM

This lists 4381 as having a point total for this event as 70... but obviously the totals they show add up to 69. And on the rankings their totals show as 69, but the event shows 70.

I'm gonna guess this 69 is right because like - it's just addition?

Reply:
Ah thank you for finding that! I thought I had just miss-typed 70 in my previous PR and it happened to work out, but yeah FIRST's calculations don't seem to add up...